src/network/network_content.cpp

```diff
@@ -804,7 +815,32 @@ void ClientNetworkContentSocketHandler::DownloadContentInfo(ContentID cid)
 	if (std::ranges::find(this->requested, cid) != this->requested.end()) return;
 
 	this->requested.push_back(cid);
-	this->RequestContentList(1, &cid);
+	this->queued.push_back(cid);
+	_request_queue_timeout.Reset();
+}
+
+/**
+ * Send a content request for queued content info download.
+ */
+void ClientNetworkContentSocketHandler::RequestQueuedContentInfo()
+{
+	if (this->queued.empty()) return;
+
+	/* Wait til we've briefly stopped receiving data (which will contain more content) before making the request. */
```

Comment:
```suggestion
	/* Wait 'til we've briefly stopped receiving data (which will contain more content) before making the request. */
```
Or `till` or `until` according to https://www.oed.com/dictionary/til_conj?tab=factsheet#18418829